src/fake_rtc.c

```diff
@@ -0,0 +1,84 @@
+#include "global.h"
+#include "string_util.h"
+#include "strings.h"
+#include "text.h"
+#include "rtc.h"
+#include "fake_rtc.h"
+
+struct Time *FakeRtc_GetCurrentTime(void)
+{
+#if OW_USE_FAKE_RTC
+	return &gSaveBlock3Ptr->fakeRTC;
```

Comment:
spaces instead of tabs (for the entire file)

Reply:
https://github.com/rh-hideout/pokeemerald-expansion/pull/4910/commits/957dda9d1f078ae2fec18f1a7eaf881795225cb6